src/flight-search/Merger.js

```diff
@@ -12,6 +12,7 @@ FlightSearchMerger.prototype = {
 
     this._mergeStaticData(response);
     this._mergeLegs(response.legs);
+    this._mergeLegConditions(response.legConditionIds)
```

Comment:
This one too, and run build then commit and all is well sir

Reply:
haha, ok boss